style/icons.css

```diff
@@ -86,3 +86,46 @@
 [data-jp-theme-light='false'] .jp-ViewBreakpointIcon {
   background-image: url('icons/md-dark/view-breakpoint.svg');
 }
+
+.jp-debugger-switch-button::before {
```

Comment:
Since those are new classes, should we try to follow the same CSS naming convention?